sentry-core/src/main/java/io/sentry/core/exception/ExceptionMechanismThrowable.java

```diff
@@ -1,26 +1,32 @@
 package io.sentry.core.exception;
 
 import io.sentry.core.protocol.Mechanism;
+import org.jetbrains.annotations.Nullable;
 
 /**
  * A throwable decorator that holds an {@link io.sentry.core.protocol.Mechanism} related to the
  * decorated {@link Throwable}.
  */
 @SuppressWarnings("serial")
 public final class ExceptionMechanismThrowable extends Throwable {
+  private static final long serialVersionUID = 142345454265713915L;
```

Comment:
we are not serializing this class, but converting to our `SentryException`, why do we need that? if so, you may remove `@SuppressWarnings("serial")`